constants/members.json

```diff
@@ -164,9 +176,9 @@
     {
       "name": "Eugene Zhang",
       "role": "developer",
```

Comment:
so Eugene is actually now technical lead for casa, could you change this?

Reply:
confirmed

Reply:
updated